sdk/eventgrid/azure-eventgrid/setup.py

```diff
@@ -51,13 +51,12 @@
         "Development Status :: 5 - Production/Stable",
         'Programming Language :: Python',
         'Programming Language :: Python :: 3 :: Only',
-        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
```

Comment:
separate PR for 3.11 and 3.12 support :)

Reply:
will make rn